src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
```

Comment:
```suggestion
  hide() {
    this.hidden = true;
  }
```
It would be confusing if method hide would unhide an animal.